pkgs/servers/metabase/default.nix

```diff
@@ -0,0 +1,28 @@
+{ pkgs, stdenv, lib, fetchurl, makeWrapper, jre }:
+
+stdenv.mkDerivation rec {
+  name = "metabase-${version}";
+  version = "0.27.2";
+
+  jar = fetchurl {
+    url = "http://downloads.metabase.com/v${version}/metabase.jar";
+    sha256 = "1xsd6k362kajbf6sw0pb7zkd686i350fqqin858a5mmjlm5jkci7";
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  phases = "installPhase";
```

Comment:
Please do not explicitly list phases, instead disable those that are causing problems. While it may not matter to this package in particular, it is a common source of bugs.